### ISSUE TITLE
cli: starting queued workflows

### ISSUE THREAD

**Issue by tiborsimko** [reanahub/reana-client#357]
Sometime the workflow is in a queued state:

```console
$ reana-client status
NAME       RUN_NUMBER   CREATED               STATUS   PROGRESS
workflow   1            2020-01-17T11:03:08   queued   -/-     
```

and, if for some reason `reana-server` cannot start it automatically -- e.g. because the message was lost; see another ticket on this -- then the end users cannot trigger the starting action via `reana-client` because of:

```console
$ reana-client start -w workflow.1
Workflow could not be started: 
Workflow cannot be started again.
```

Currently, only cluster admins can unblock the situation by setting workflow status to `created` and issuing the `start` command.

What we could do is to allow end users to do `start` command and on the reana-server endpoint side check the queue and if everything is OK and the job is properly waiting for execution, then do nothing (=as is the case now); however if there is something wrong and the job is still queued even though it could run, the reana-server endpoint could accept the `start` command (=new behaviour).

**Comment by tiborsimko** [comment]
"see another ticket" above is https://github.com/reanahub/reana/issues/253

**Comment by diegodelemos** [comment]
> and, if for some reason reana-server cannot start it automatically -- e.g. because the message was lost; see another ticket on this -- then the end users cannot trigger the starting action via

This doesn't happen anymore, see https://github.com/reanahub/reana/issues/253.

I think that what is left in this ticket is to give a better message, saying something like "your workflow is already queued for execution" or similar.
